packages/contents/src/drivefs.ts

```diff
@@ -425,7 +425,7 @@ export class DriveFSEmscriptenNodeOps implements IEmscriptenNodeOps {
   };
 
   readlink = (node: IEmscriptenFSNode | IEmscriptenStream): string => {
-    throw new this.fs.FS.ErrnoError(this.fs.ERRNO_CODES['EPERM']);
+    throw new this.fs.FS.ErrnoError(this.fs.ERRNO_CODES['EINVAR']);
```

Comment:
Sorry for the wrong error number, this makes much more sense!

Long term (if there is a need for it?) we should have JupyterLab drives support symlinks for real.